packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,37 @@
+## Guidelines
+
+### General
+
+-   **Use links**, instead, to **redirect users to another page** of the current website or app, or to another website.
+-   Avoid using too many buttons on a page. If many buttons are to be added, **less popular** or **less important** actions may be **grouped in a “more” button** (dropdown list) or **visually styled as isolated actions or links**.
+
+### Labels
+
+-   The button **label** should allow to **foresee what will happen** when clicking it.
+-   Button **labels** should be **short** and use **“trigger words”**.
+-   As much as possible, the **first word of the label** should be an **action verb** (e.g., the label for a button that allows the user to add a filter could be “Add Filter").
+
+### Alterative Texts
+
+-   When a **button includes only an icon** (no label), the **label** should be added in the **alternative text element in the HTML** and appear in a **tooltip** when the cursor is above the button.
+-   An **icon with a label does not need alternative text**. In this case, the icon is a visual help to identify the purpose faster and filling it will cause a screen reader to repeat the text twice.
+
+## Variations
+
+The Button Type choice is defined by the function of the button (e.g. primary or a secondary action), the number of actions available in the screen (e.g. only one, two or more actions) and its location (e.g. displayed in a header, action bar, page or footer).
```

Comment:
Suggestion:
The button type choice depends on the function of the button (e.g., primary action, secondary action), the number of actions available in the screen (e.g., one, two, several), and its location (e.g., header, action bar, page, footer).